srcpkgs/librime/template

```diff
@@ -1,17 +1,19 @@
 # Template file for 'librime'
 pkgname=librime
-version=1.5.3
-revision=4
+version=1.7.2
```

Comment:
1.7.3 is out

Reply:
Is that not a pre-release version?  
https://github.com/rime/librime/releases/tag/1.7.3

Reply:
It says pre-release, but it also includes only two commits fixing undefined behavior. So I'd prefer that we move to it. Might make sense to ask upstream why they tagged it that way.

Reply:
The second one (buffer overflow) was in a plugin that we don't have. (do we in anyway pull that plugin in?)
The first one is near-null-pointer dereference, I think.